packages/main/src/MultiComboBox.ts

```diff
@@ -674,6 +674,7 @@ class MultiComboBox extends UI5Element {
 
 	_onkeydown(e: KeyboardEvent) {
 		const isArrowDownCtrl: boolean = isDownCtrl(e);
+		const isCtrl = !!(e.metaKey || e.ctrlKey);
```

Comment:
event.metaKey and event.ctrlKey are boolean by default you don't need to cast them as such so just 
`const isCtrl = e.metaKey || e.ctrlKey;` should be enough

you could also at a type as the variable above it